### DISCUSSION TITLE
How to install

### DISCUSSION THREAD

**Discussion by Ademola101** [JedWatson/react-select#5372] [Ideas]
It will be great if the how to install instructions is on the Getting started page

**Comment by lukebennett88** [comment]
Thanks for the suggestion @Ademola101.
We're planning to improve the docs pretty soon.
In the meantime, if you didn't know already, there are instructions [in the README](https://github.com/jedwatson/react-select#installation-and-usage).

**Reply by Rall3n** [reply]
@lukebennett88 @Ademola101 Installation instructions are already part of the ["Getting started"](https://react-select.com/home#getting-started) section in the documentation.

**Reply by lukebennett88** [reply]
I took it at face value that the instructions weren't there — they totally are. Good call out @Rall3n 🤦‍♂️
